vertical-pod-autoscaler/pkg/admission-controller/resource/vpa/handler.go

```diff
@@ -134,6 +134,12 @@ func validateVPA(vpa *vpa_types.VerticalPodAutoscaler, isCreate bool) error {
 					return fmt.Errorf("max resource for %v is lower than min", resource)
 				}
 			}
+			ControlledValues := policy.ControlledValues
+			if mode != nil && ControlledValues != nil {
+				if *mode == vpa_types.ContainerScalingModeOff && *ControlledValues == vpa_types.ContainerControlledValuesRequestsAndLimits {
+					return fmt.Errorf("Mode can not be Off when LimitScaling is Auto")
```

Comment:
I think it's more the other way round:
ControlledValues shouldn't be specified if container scaling mode is off.

Reply:
Fixed